### ISSUE TITLE
How to create a "NonBlockingConnectionPool "using a "java.sql.Connection" object already present ?

### ISSUE THREAD

**Issue by ArijitCode** [davidmoten/rxjava2-jdbc#216]
I tried creating a connectionpool using rxjava2-jdbc using a java.sql.Connection Object . 
 
NonBlockingConnectionPool pool =
                Pools.nonBlocking()
                    .maxPoolSize(Runtime.getRuntime().availableProcessors() * 5)
                    .connectionProvider(ConnectionProvider.from(connection))
                    .build();
But it strangely throws this NoSuchMethodError exception :
 
Caused by: java.lang.NoSuchMethodError: com.github.davidmoten.guavamini.Preconditions.checkNotNull(Ljava/lang/Object;Ljava/lang/String;)Ljava/lang/Object

**Scenario :- 
I already have a   java.sql.Connection Object , How to I use that to create a org.davidmoten.rx.jdbc.Database Object so that I can execute my SQL queries from ?**

**Comment by davidmoten** [comment]
Ah, there was a breaking change in guava-mini, check you have latest version of rxjava-jdbc and other libs. If there is a mismatch I'll fix with another release.

**Comment by ArijitCode** [comment]
yeah , will upgrade , but What I really would need now is this :- 

**I already have a java.sql.Connection Object , How to I use that to create a org.davidmoten.rx.jdbc.Database Object so that I can execute my SQL queries from ?**

Can you please help me with some sample code doing specifically the above ?
